owners/OWNERS.example

```diff
@@ -131,11 +131,11 @@
   ],
 
   // The root-level owners file--and ONLY the root-level owners file--may
-  // specify an optional `reviewerTeam` property, providing the name of a GitHub
+  // specify an optional `reviewerPool` property, providing the name of a GitHub
   // team. If present, the owners check will require that at least one member of
   // this team has given approval. It is possible that one reviewer is both a
   // member of this reviewer team and provides owners coverage.
-  reviewerTeam: 'ampproject/reviewers-amphtml'
+  reviewerPool: ["ampproject/reviewers-amphtml", "renovate-approve"]
```

Comment:
cosmetic nit: The actual bot reviewer account we'll add to the owners file in `amphtml` is going to be `renovate-bot`, and not `renovate-approve`. (Might make sense to have this example reflect the real name we'll use.)

Reply:
```suggestion
  reviewerPool: ["ampproject/reviewers-amphtml", "approver-bot"]
```

Reply:
I've made it a more generic bot name instead, since this is not meant to be tied to any specific bot (or repo, even though there are a lot of amphtml references)